packages.yml

```diff
@@ -1,3 +1,3 @@
 packages:
   - package: fishtown-analytics/dbt_utils
-    version: [">=0.4.0", "<0.6.0"]
```

Comment:
I guess this _would_work with earlier versions, might would return errors. I'd prefer to be tightly scoped here instead

Reply:
Agreed! If we're upgrading a package to use functionality only available in dbt >= 0.18.0 (`dispatch`), I think we should also scope it to require utils >= 0.6.0.

If a package does _not_ require net-new functionality, though (`stitch_utils`, `logging`, `redshift`), I don't know that we need to raise the lower bounds + cut a minor version?